plugins/module_utils/rds.py

```diff
@@ -744,6 +745,26 @@ def describe_db_cluster_parameter_groups(
     return result
 
 
+@AWSRetry.jittered_backoff()
+def describe_db_parameter_groups(
```

Comment:
You also have to return tags information.

Reply:
updated code for getting tags